src/dialects/sqlite3/index.js

```diff
@@ -39,6 +39,8 @@ assign(Client_SQLite3.prototype, {
 
   TableCompiler: TableCompiler,
 
+  SqlString: require('../../query/string'),
```

Comment:
Shouldn't be necessary to set here since client base implementation already does that.